docs/user-guide/data-reduction.md

```diff
@@ -0,0 +1,141 @@
+# Reflectometry data reduction procedure
+
+The goal of the reflectometry data reduction is to compute the sample reflectivity $R(Q)$ as a function of the momentum transfer $Q$.
+
+## Preliminaries
+
+The detector data consists of a list $EV$ of neutron detector events.
+For each neutron event in the event list we know what wavelength $\lambda$ it had and we know the pixel number $j$ of the detector pixel that it hit.
+The detector pixel positions are known and so is the position of the sample and the orientation of the sample.
+From that information we can compute the reflection angle (assumed equal to the incidence angle) $\theta$, and the momentum transfer $Q$ caused by the interaction with the sample.
+
+The purpose of this text is not to describe how the event coordinates $Q$ and $\theta$ are derived from the raw event data and the geometry information, so for now just take those relations for given.
+
+To avoid overcomplicating the description it is assumed that the sample- and reference measurements were made over the same length of time, and it is assumed the neutron intensity from the source did not vary between the two measurements.
+
+## Model
+
+The sample reflectivity is related to the intensity of neutron counts in the detector by the model
+
+$$
+I_{sam}(\lambda, j) = F(\theta(\lambda, j, \mu_{sam}), w_{sam}) \cdot R(Q(\lambda, \theta(\lambda, j, \mu_{sam}))) \cdot I_{ideal}(\lambda, j)
+$$ (model)
+where $I_{sam}(\lambda, j)$ represents the number of neutrons detected in the $j$ pixel of the detector having a wavelength in the interval $[\lambda, \lambda + d\lambda]$. $I_{ideal}$ represents the number of neutrons that would have been detected if the sample was a perfect reflector and large enough so that the footprint of the focused beam on the sample was small compared to the sample. $F(\theta, w)$ is the fraction of the focused beam that hits the sample. It depends on the incidence angle $\theta$ and on the size of the sample represented by $w$. $\mu_{sam}$ is the sample rotation.
```

Comment:
$\lambda$ would typically be reported as a bin-centre. Here, it is the lower edge. This is in principle fine. But it may lead to confusion between the maths here and actual data.

Reply:
Good point!